pkg/cmd/get/test_data/get_build_logs/tekton_build_logs/pipelinerun.yml

```diff
@@ -8,7 +8,7 @@ metadata:
     created-by-prow: "true"
     owner: fakeowner
     prowJobName: 3930a5ce-9da8-11e9-9b3d-acde48001122
-    repo: fakerepo
+    repository: fakerepo
```

Comment:
I guess, this then surfaced after my latest change.

Reply:
Yup!